### DISCUSSION TITLE
Is there something like Split View, where you can view and compare two images next to each other in the same window/instance?

### DISCUSSION THREAD

**Discussion by ProtoBelisarius** [nomacs/nomacs#1014] [Q&A] [answered]
Geeqie can split the window into 2 or 4 segments where you can compare different images, zoom into them seamlessly and open them from different directories. i find this extremely useful but I like nomacs a lot and would switch if it could show at least 2 at the same time. So I wondered if there is a feature that can sort of emulate that. Maybe a plugin? I havent found anything like that googling. Are there maybe plans to add this feature or is this not something to be considered and wont be added in the future?

**Comment by nyncral** [comment, marked as answer]
since this is the second result on duckduck go I'm posting this link here : 
https://nomacs.org/blog/synchronization/
